tests/uc_oss/test_uc_oss_integration.py

```diff
@@ -0,0 +1,161 @@
+import os
+import subprocess
+import sys
+
+import pandas as pd
+import pytest
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+import mlflow
+
+from tests.tracking.integration_test_utils import _await_server_up_or_die
+
+
+@pytest.mark.skipif(
+    "UC_OSS_INTEGRATION" not in os.environ,
+    reason="This test is only valid w/in the github workflow integration job",
+)
+@pytest.fixture(scope="module")
+def setup_servers():
```

Comment:
Can we also run the UC server in this fixture?

Reply:
I don't think so (at least not easily).  It looks like Ben and Serena ran into the same problem in the UC OSS repo with their tests.  They also spin up a server from the github workflow definition instead of from w/in the python test, probably because of the dependency issue I was seeing with trying to execute from w/in python.

Reply:
Tested my suggestion here: https://github.com/mlflow/mlflow/pull/14186

Reply:
@kriscon-db we do it at the suite level in UC OSS because we're testing multiple integration packages. What @harupy recommends here is definitely optimal for MLflow :)

Reply:
Gotcha.  Looks like the missing ingredient was leaving the build step in.